tsdef/AudioTrack.d.ts

```diff
@@ -0,0 +1,13 @@
+import { Track } from './Track';
+
+export class AudioTrack extends Track {
+  isStarted: boolean;
+  isEnabled: boolean;
+  kind: 'audio';
+  mediaStreamTrack: MediaStreamTrack;
+
+  _attachments?: HTMLMediaElement[];
```

Comment:
This seems like it shouldn't be exposed, since it's a private member.

Reply:
I grabbed this from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/twilio-video/index.d.ts) repo which I believe is what we wanted?

Reply:
+1 to what @ryan-rowland said, we should not document private members.

Reply:
For some context, looks like Definitely Typed added this because of this comment: https://github.com/twilio/twilio-video.js/issues/294#issuecomment-389708981

But it looks like that comment is in reference to version 1.x.x of the SDK, so I think we can remove this private member.